build/build_android_llm_demo.sh

```diff
@@ -91,12 +91,18 @@ build_aar() {
   popd
 }
 
-build_android_llm_demo_app() {
+build_android_demo_apps() {
```

Comment:
Should we rename the script and steps in android-perf as well?

Reply:
Yes! Let's do that later. It's really used in different places